brands/office/energy_supplier.json

```
@@ -57,5 +57,34 @@
       "name": "Naturgy",
       "office": "energy_supplier"
     }
+  },
+  "office/energy_supplier|東京電力": {
+    "countryCodes": ["jp"],
+    "matchNames": ["東電"],
+    "tags": {
+      "brand": "東京電力",
+      "brand:en": "TEPCO",
+      "brand:ja": "東京電力",
+      "brand:wikidata": "Q11284512",
```

Your wikidata tag is wrong. Is this one right?

```suggestion
      "brand:wikidata": "Q333894",
```

---

Nice to review this. I fixed.